demos/demo_reactive_planners_solo12_step_adjustment_walk.py

```diff
@@ -0,0 +1,317 @@
+"""Demo file for running the reactive planner on SOLO 12
```

Comment:
Not it's fine here.
I would just write a small main at the bottom of the file to run 1 reversion of the demo

Reply:
Ok, I can write a `main()` function. But as @Lhumd suggested, could I move this file to the python/reactive_planners/demos folder so that CMake will `install()` the script so I can use it in another package?

Reply:
Ah if you want this demo to be part of the public API (other package can use it) then you must move it to the python folder.
But then find another name than `demo` something saying it's specific to solo and that it's a controller.

Reply:
In the main function, you want me to run the code in pybullet. Is that correct?